source/_introduction.html.md.erb

```diff
@@ -298,9 +298,37 @@ The JSON representation of the objects returned from the List All REST endpoints
 Smartsheet will add new functionality and bug fixes to the API over time. Make sure that your code can handle new JSON properties gracefully. 
 Also, make sure your code does not depend on the order in which JSON objects are returned, unless it is explicitly stated in this documentation. 
 
+In some cases, such as when new column types or new widgets are added, Smartsheet offers a backwards-compatibile option as well as giving you access to the new functionality. For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response:
+
+* As text, such as column type TEXT_NUMBER
```

Comment:
When the level does not support new Grid data type, the column will ALWAYS be converted to TEXT_NUMBER